tests/functional/generate_pages.py

```diff
@@ -69,6 +69,13 @@
 
 """
 
+RELATIONSHIP_RADIO_CLICKER = r"""  click{optionName}() {
+    browser.element('[id="{optionId}"]').click().pause(300)
```

Comment:
We change the element to have an "animating" class or something while it is animated but the 300ms isn't as arbitrary as you think as it's the actual time in the animation in the css https://github.com/ONSdigital/eq-survey-runner/blob/master/app/assets/styles/partials/components/_relationship.scss#L38